backend/gameplay/models.py

```
@@ -200,6 +200,10 @@ def sync_get_player_allowed_to_move(self, white_player_username, black_player_us
 		else:
 			return black_player_username
 		
+	@database_sync_to_async
```

Pylint (3.3.7)

[warning] Bad indentation. Found 1 spaces, expected 4

(W0311)